home/jobs/BIOFORMATS-push/config.xml

```diff
@@ -16,9 +33,9 @@
           <description></description>
           <choices class="java.util.Arrays$ArrayList">
             <a class="string-array">
+              <string>none</string>
```

Comment:
Is this reordering made on purpose ? This will set `none` as the default Travis status to use which might have some unwanted effect.